components/form-builder/panel/ElementPanel.tsx

```diff
@@ -466,29 +470,58 @@ const ElementPanelDiv = styled.div`
 export const ElementPanel = () => {
   const {
     form: { elements },
-    add,
   } = useTemplateStore();
 
-  if (!elements.length) {
-    return (
-      <button
-        style={{ marginBottom: 20 }}
-        className="gc-button gc-button--secondary"
-        onClick={() => {
-          add();
-        }}
-      >
-        Add form element
-      </button>
-    );
-  }
+  const introTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "Add an introduction" }],
+    },
+  ];
+
+  const confirmTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "Thank you for participating in the ice cream survey!" }],
+    },
+  ];
+
+  const policyTextPlaceholder = [
+    {
+      type: "paragraph",
+      children: [{ text: "All email addresses will be not be shared." }],
+    },
+  ];
 
   return (
     <ElementPanelDiv>
+      <RichTextLocked
+        addElement={true}
+        initialValue={introTextPlaceholder}
+        schemaProperty="introduction"
```

Comment:
Do we add these later to the schema, or do we not need to just yet?

Reply:
Will update the schema on we get the saving / reloading working in a follow-up PR once we sort out the editor.